aqi_monitor/apps/monitor/tasks.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, unicode_literals
+
+import requests
+import json
+
+from celery.schedules import crontab
+from aqi_monitor.celery import app
+from . import parse_aqi_data
+
+
+app.conf.beat_schedule = \
+    {"send_message": {
+        "task": "apps.monitor.tasks.get_aqi",
+        "schedule": crontab(minute="*/1"),
```

Comment:
I think we can change task execution interval  as you've realized that task scheduling works as it has to.

Reply:
Fixed